### PR TITLE
pcm-596

### PR DESCRIPTION
**Pull request by sshumake**
@renujhamtani @pbathia

### CODE REVIEW

File: app/cases/controllers/addCommentSection.js [hunk 1]
```diff
@@ -88,7 +71,7 @@ angular.module('RedhatAccess.cases').controller('AddCommentSection', [
                 $scope.addingComment = false;
                 $scope.progressCount = 0;
             };
-            if(!CaseService.disableAddComment){
+            if(!CaseService.disableAddComment && CaseService.commentText !== 'undefined'){
```
Comment (pbathia) [inline app/cases/controllers/addCommentSection.js]: I didn't understand the purpose for adding this condition, would you please explain the scenario.

Comment (sshumake) [inline app/cases/controllers/addCommentSection.js]: The same button submits attachments and adds comments.  I was running into a scenario where when only an attachment was being added if the comment field was blank a blank comment was being attached resulting in an error.

Comment (sshumake) [inline app/cases/controllers/addCommentSection.js]: @pbathia ^

Comment (pbathia) [inline app/cases/controllers/addCommentSection.js]: ok got it Spenser. Thanks..

**Comment by renujhamtani** [comment]
Reviewed!Merging!

**Pull request merged**